pynxtools/dataconverter/readers/xrd/config.py

```diff
@@ -0,0 +1,117 @@
+"""This is config file that mainly maps nexus definition to data path in raw file."""
```

Comment:
very similar to, but not the same as Lucas' new solution for supporting different file format mappings. Have you tried to harmonise your solutions?

Reply:
Do you mean somthing like pynxtools/dataconverter/readers/xps/config_file_sle.json? @lukus would you make a comment?

Reply:
I have used one config file for each solution (see [here](https://github.com/FAIRmat-NFDI/pynxtools/pull/188/files)) and then the selected parser automatically uses the correct config file. I chose to do it that way because I plan to also have a metadata parser (possibly one for each main parser) and then I would like to resolve in the config file automatically if the data is provided by the metadata parser or from the ELN, like so:
```json
{
   "/ENTRY[entry]/SAMPLE[sample]/gas_pressure":[
      "@metadata:gas_pressure",
      "@eln"
   ]
}
```
In that case, having just one config file for all readers seems to complicate things.

Reply:
This is a nice approach, but I implemented it differently. Mainly reader will read the data from input file, then from eln. But not all the data will read from eln for example NXdata or name of definition. So without any annotation of @eln I have integrated it with some mantaining function.